mmv1/templates/terraform/examples/scc_v2_project_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,12 @@
+resource "google_project" "default" {
+  name       = "My Test Project"
+  project_id = "<%= ctx[:vars]['project_id'] %>"
+}
+
```

Comment:
This project will most likely also need an iam_member resource allowing the mute config to be created.

Reply:
addressed